Rainbow/rainbow.css

```diff
@@ -0,0 +1,24 @@
+.rainbow {
+  background: -webkit-linear-gradient(left, red, orange , yellow, green, cyan, blue, violet);
+  background: -o-linear-gradient(right,red,  orange, yellow, green, cyan, blue, violet);
+  background: -moz-linear-gradient(right,red,  orange, yellow, green, cyan, blue, violet);
+  background: linear-gradient(to right,red,  orange , yellow, green, cyan, blue, violet);
```

Comment:
Slightly wonky spacing here in the parentheses

Reply:
Addressed in #b865de65